.github/workflows/build.yml

```diff
@@ -0,0 +1,128 @@
+name: 'Build & Test'
+on:
+   push:
+     branches:
+       - main
```

Comment:
main branch is `master` on this repo.

We can merge and I can change it to main though